### PR TITLE
graphql endpoints for invitation

### PR DESCRIPTION
**Pull request by dipendraupreti**
…tion

### CODE REVIEW

File: packages/user/src/model/invitations/resolver.ts [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+    const { app, config, database, dbSchema, reply } = context;
+
+    const { token, data } = arguments_;
+
+    try {
+      const { email, password } = data;
+
+      //  check if the email is valid
+      const emailResult = validateEmail(email, config);
+      if (!emailResult.success && emailResult.message) {
```
Comment (rameshlohala) [inline packages/user/src/model/invitations/resolver.ts:49]: missing empty line before if statement

File: packages/user/src/model/invitations/resolver.ts [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+        const mercuriusError = new mercurius.ErrorWithProps(
+          emailResult.message
+        );
+
+        return mercuriusError;
+      }
+
+      // password strength validation
+      const passwordStrength = validatePassword(password, config);
+      if (!passwordStrength.success && passwordStrength.message) {
```
Comment (rameshlohala) [inline packages/user/src/model/invitations/resolver.ts:59]: missing empty line before if statement

File: packages/user/src/model/invitations/resolver.ts [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+          "Email do not match with the invitation"
+        );
+
+        return mercuriusError;
+      }
+
+      // signup
+      const signUpResponse = await emailPasswordSignUp(email, password);
+
+      if (!(signUpResponse.status === "OK")) {
```
Comment (rameshlohala) [inline packages/user/src/model/invitations/resolver.ts:96]: can't it be just 

` if (signUpResponse.status !== "OK") {`  ?

Comment (dipendraupreti) [reply packages/user/src/model/invitations/resolver.ts:96]: yes, it can be.

**Pull request merged**